docs/how-to-guides/monitoring-with-prometheus-operator.md

```diff
@@ -15,6 +15,13 @@
   * [Accessing Grafana](#accessing-grafana)
     * [Using port forward](#using-port-forward-1)
     * [Using Ingress](#using-ingress-1)
+* [Add custom Grafana dashboards](#add-custom-grafana-dashboards)
```

Comment:
Should it be `Add` or `Adding`. It seems we are mixing the forms here (e.g. `Accessing` vs `Deploy`).
```suggestion
* [Adding custom Grafana dashboards](#add-custom-grafana-dashboards)
```

Reply:
Fixed it to the simple present tense in the doc. PTAL at the second commit.